webapp/components/interop.js

```diff
@@ -20,23 +19,31 @@ import PARAMS_BY_YEAR from '../static/interop-data.json' assert {type: 'json'};
 class InteropDataManager {
   constructor(year) {
     // prepare all year-specific info for reference.
-    this.year = year;
-    const yearInfo = PARAMS_BY_YEAR[year];
-    this.focusAreas = yearInfo.focus_areas;
-    this.summaryFeatureName = yearInfo.summary_feature_name;
-    this.csvURL = yearInfo.csv_url;
-    this.tableSections = yearInfo.table_sections;
-    this.prose = yearInfo.prose;
-    this.matrixURL = yearInfo.matrix_url;
-    this.issueURL = yearInfo.issue_url;
-    this.investigationScores = yearInfo.investigation_scores;
-    this.investigationWeight = yearInfo.investigation_weight;
-
+    this.fetchYearData(year)
```

Comment:
Similar to the race condition between `fetchYearData` and `fetchYearData`, we don't want a race condition when `InteropDataManager`'s value is assigned to the view class and when this `fetchYearData` is fetched.  We should somehow update view after this promise succeeds, through observable values, or use `await`, e.g. await `this.dataManager.getDataTable(feature, stable)` to make it synchronous

Reply:
I was not able to reproduce your error above, but I'm pretty sure I understand where it's coming from. All of these methods await `_dataLoaded`, which was set when both `_loadCsv` invocations were finished, but these invocations need to run after the year info is obtained. I'm not certain the best way to verify this since I wasn't able to reproduce, but I've added a change that handles each of these calls sequentially as intended.